src/frontend/admin/src/components/templates/courses/form/CourseForm.tsx

```diff
@@ -132,6 +142,13 @@ export function CourseForm({ course, shortcutMode = false, ...props }: Props) {
                     label={intl.formatMessage(courseFormMessages.codeLabel)}
                   />
                 </Grid>
+                <Grid xs={12} md={6}>
+                  <RHFTextField
+                    type="number"
+                    name="effort"
+                    label={intl.formatMessage(courseFormMessages.effortLabel)}
+                  />
+                </Grid>
```

Comment:
We should mention which unit is expected (hour).

I think an help text should do the job